homeassistant/components/cover/zwave.py

```diff
@@ -110,24 +110,42 @@ class ZwaveGarageDoor(zwave.ZWaveDeviceEntity, CoverDevice):
     def __init__(self, values):
         """Initialize the zwave garage door."""
         ZWaveDeviceEntity.__init__(self, values, DOMAIN)
+        self._state = None
         self.update_properties()
 
     def update_properties(self):
         """Handle data changes for node values."""
         self._state = self.values.primary.data
+        _LOGGER.debug("self._state=%s", self._state)
+
+    @property
+    def is_opening(self):
+        """Return true if cover is in an opening state."""
+        if self._state == "Opening":
```

Comment:
Small nitpick, here we can just write 
```python
return self._state == "Opening"
```
since the equality operator already returns a boolean. Same for the others.